frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts

```diff
@@ -26,24 +26,24 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component, OnDestroy, Output, EventEmitter} from '@angular/core';
+import {Component, EventEmitter, Output} from '@angular/core';
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
-import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
 import {WorkPackageCacheService} from "app/components/work-packages/work-package-cache.service";
 import {Subject} from "rxjs";
 import {debounceTime, distinctUntilChanged, map, tap} from "rxjs/operators";
 import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {input} from "reactivestates";
 import {QueryFilterResource} from "core-app/modules/hal/resources/query-filter-resource";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts#L40"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>